app/javascript/index_books.vue

```diff
@@ -0,0 +1,31 @@
+<template>
```

Comment:
The name of the file is not really good. Better one would be `books_list`

Reply:
👍